doc/api/dgram.md

```diff
@@ -299,6 +299,62 @@ added: v0.6.9
 Sets or clears the `SO_BROADCAST` socket option.  When set to `true`, UDP
 packets may be sent to a local interface's broadcast address.
 
+### socket.setMulticastInterface(multicastInterface)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `multicastInterface` {String}
+
+Sets the default outgoing multicast interface of the socket to a chosen interface or back to system interface selection. The `multicastInterface` must be a valid string representation of an IP from the socket's family.
+
+For IPv4 sockets, this should be the IP configured for the desired physical interface. All packets sent to multicast on the socket will be sent on the interface determined by the most recent successful use of this call.
+
+For IPv6 sockets, `multicastInterface` should include a Scope ("IP%Scope") to indicate the interface as in the examples that follow. In IPv6, individual send calls can also use explicit Scope in addresses, so only packets sent to a multicast address without specifying an explicit Scope are affected by the most recent successful use of this call.
+ 
+#### Examples: IPv6 Outgoing Multicast Interface
+On most systems, where Scope format uses the interface name:
```

Comment:
Hmm, this documents is writing for Node.js user who might not be too familiar with this even at operating system and the libuv.

Basically I think here we should notice reader that:

```md
Note: all `Scope` in this section represents the [IPv6 address scope](https://en.wikipedia.org/wiki/IPv6_address#IPv6_address_scopes)
```

Then any confuser will read the wiki if he wants to know :-)